podtato-head-monolith/docker/DockerfileV0.1.0

```diff
@@ -0,0 +1,34 @@
+FROM library/golang:latest AS builder
+
+# Set necessary environment variables needed for our image
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+# Move to working directory /build
+WORKDIR /build
+
+# Copy and download dependency using go mod
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+# Copy the code into the container
+COPY . .
+
+# Build the application
+RUN go build -o podtatoserver ./podtatoserver.go
+
+# Build a small image
+FROM scratch
+
+COPY --from=builder /build/podtatoserver /
+COPY /static  /static
+COPY /static/podtato-new.html  /podtato-new.html
```

Comment:
The HTML file is copied to `/podtato-new.html` but the script still uses `/static/podtato-new.html`. The line could therefore be deleted.

Reply:
Removed this in the latest commit.